code/game/objects/items/weapons/table_rack_parts.dm

```diff
@@ -62,7 +62,7 @@
 			Grass.amount -= 1
 		else
 			del(Grass)
-		new /obj/item/weapon/table_parts/wood/poker( src.loc )
+		new /obj/item/weapon/table_parts/wood/poker( user.loc )

```

Comment:
I was going to say mechs, but you can't use hands in those. I suppose it's just a contingency for if humans are ever placed in nullspace and allowed to use their hands, but if that doesn't exist, then what you wrote is fine.